src/clusterfuzz/_internal/bot/tasks/commands.py

```diff
@@ -276,7 +276,8 @@ def process_command_impl(task_name,
       logs.log_error(error_string)
       raise errors.BadStateError(error_string)
 
-    job_queue_suffix = tasks.queue_suffix_for_platform(job.platform)
+    job_queue_suffix = tasks.queue_suffix_for_platform(
```

Comment:
nit: can we rename this to `job_base_queue_suffix` for consistency?

Reply:
Done.